extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -496,6 +471,65 @@ public function getArticle() {
 		$this->response->setVal( 'data', $data );
 	}
 
+	public function getArticle() {
```

Comment:
mark this function as transitional one and it should be removed after we are 100% transitioned to new syntax